spec/iex/resources/logo_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe IEX::Resources::Logo do
+  context "retrieve company's logo", vcr: { cassette_name: 'logo/msft' } do
+    subject do
+      IEX::Resources::Logo.get('MSFT')
+    end
+
+    it 'retrieves a logo' do
+      expect(subject['url']).to eq 'https://storage.googleapis.com/iex/api/logos/MSFT.png'
```

Comment:
I missed that it's returning a structure. This should actually return a 1st class object called `Logo` with a property called `url`. See how it's done for OHLC.

Reply:
Good catch! I didn't notice that. So, now it should be good:

```ruby
    9:     it 'retrieves a logo' do
   10:       require 'byebug'
   11:       byebug
=> 12:       expect(subject.url).to eq 'https://storage.googleapis.com/iex/api/logos/MSFT.png'
   13:     end
   14:   end
   15: end
(byebug) subject
#<IEX::Resources::Logo url="https://storage.googleapis.com/iex/api/logos/MSFT.png">
(byebug) subject.url
"https://storage.googleapis.com/iex/api/logos/MSFT.png"
(byebug)
```